app/src/screens/ActiveCall/hooks.ts

```diff
@@ -108,76 +106,86 @@ const useHangup = (
   return { handle, isDisabled };
 };
 
-const matchOnAudioDevices = (
-  audioDevices: AudioDeviceInfo[],
-  selectedDevice: AudioDeviceInfo | null,
-  dispatch: Dispatch,
-) => {
-  const earpieceDevice = audioDevices.find(
-    (dev) => dev.type === TwilioAudioDevice.Type.Earpiece,
-  );
-
-  const speakerDevice = audioDevices.find(
-    (dev) => dev.type === TwilioAudioDevice.Type.Speaker,
-  );
-
-  return match<
-    [typeof earpieceDevice, typeof speakerDevice, typeof selectedDevice],
-    [() => void, boolean, boolean]
-  >([earpieceDevice, speakerDevice, selectedDevice])
-    .with(
-      [P.not(undefined), P._, null],
-      [P.not(undefined), P._, { type: TwilioAudioDevice.Type.Speaker }],
-      ([{ uuid }]) => [
-        () => {
-          dispatch(selectAudioDevice({ audioDeviceUuid: uuid }));
-        },
-        false,
-        true,
-      ],
-    )
-    .with(
-      [P._, P.not(undefined), null],
-      [P._, P.not(undefined), { type: TwilioAudioDevice.Type.Earpiece }],
-      ([_, { uuid }]) => [
-        () => {
-          dispatch(selectAudioDevice({ audioDeviceUuid: uuid }));
-        },
-        false,
-        false,
-      ],
-    )
-    .otherwise(() => [() => {}, true, false]);
+/**
+ * Return type of the audio device hook.
+ */
+type AudioDevicesHookReturnType = {
```

Comment:
Too verbose and the name does not really describe the members of this type. How about `AudioDevicesHook` or `AudioDevicesHandlers`?